create_package.py

```diff
@@ -407,27 +453,32 @@ def main(
 
     has_client_code = bool(ADDON_CLIENT_DIR)
     if has_client_code:
-        client_dir: str = os.path.join(CLIENT_ROOT, ADDON_CLIENT_DIR)
+        if ADDON_CLIENT_DIR:
+            client_dir: str = os.path.join(CLIENT_ROOT, ADDON_CLIENT_DIR)
+        else:
+            client_dir = CLIENT_ROOT
         if not os.path.exists(client_dir):
-            raise RuntimeError(
+            msg = (
```

Comment:
Why?

Reply:
https://docs.astral.sh/ruff/rules/raw-string-in-exception/